src/scripts/components/shapes.es6

```diff
@@ -24,7 +24,7 @@ module.exports = class Shapes extends Component {
 
 	injectSprite() {
 		$.get(this.data.url, (response, status) => {
-			if (status == 'success') {
+			if (status === 'success') {
```

Comment:
Podle mě je `===` daleko čitelnější, protože prostě vím, že někdo myslel _„prostě to samý“_. Oproti tomu u `==` zbytečně přemýšlím, co se tam na pozadí děje za magii, jestli třeba něco nějak nemůže být třeba `undefined` nebo tak. Jistě, u stringů to je jedno, ale to, že je to jedno, si člověk uvědomí až když se nad tím zamyslí, což při užití `===` ani nemusí. Zároveň myslím, že tam funguje určitá synergie, kdy konzistentní používání `===` činí užití `==` daleko čitelnějším, protože je hned jasné, že je to záměr a že se tam děje nějaká žádoucí konverze, a i proto bych v těchto případech preferoval `===`.